blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Blockchain Basics- An Introduction to Web3 Terms and Concepts with Celo
+description: In this article we will be leanring about the basics of the blockchain and the technologies around it such as NFT, DAO, DEFI and also about the Web3 with Celo.
```

Comment:
@Ishanpathak1 One small change, can you decrease the description size to aroung 18/20 works. Larger description sized make the cards look weired in the website.

please make the same changes in users.txs

Reply:
I have made the changes